app/controllers/application_controller.rb

```diff
@@ -221,7 +221,7 @@ def invalid_auth_token(_exception)
       user_signed_in: user_signed_in?,
     )
     flash[:error] = t('errors.invalid_authenticity_token')
-    redirect_back fallback_location: new_user_session_url, allow_other_host: false
+    redirect_back fallback_location: new_user_session_url
```

Comment:
Should we want to keep these?

Reply:
:[ my merge must have gone wrong... thanks for checking

Reply:
Yeah I think I pulled from the staging SHA before it redeployed today, just redid with the staging sha (6648d2a2543b488022f642fb4cda7b4e7026fe3a) which does have that change included